newIDE/app/src/EventsSheet/index.js

```diff
@@ -362,6 +367,22 @@ export default class EventsSheet extends React.Component<Props, State> {
     return newEvents;
   };
 
+  openEventTextDialog = () => {
+    const selectedEvents = getSelectedEvents(this.state.selection);
+    if (!selectedEvents.length) return;
```

Comment:
There is a risk the last select event is not a comment/group, so better filter them (see `filterEditableWithEventTextDialog` in a next comment):

```
const editableEvents = filterEditableWithEventTextDialog(getSelectedEvents(this.state.selection));
if (!editableEvents.length) return;

const event = editableEvents[editableEvents.length - 1];  // Get the last selected event.
```